tests/storage/storage.py

```diff
@@ -43,3 +44,31 @@ def live_storage_migration_then_come_back(vm, prov_host, prov_sr, dest_host, des
     wait_for(lambda: vm.all_vdis_on_sr(prov_sr), "Wait for all VDIs back on provenance SR")
     wait_for(lambda: vm.is_running_on_host(prov_host), "Wait for VM to be running on provenance host")
     vm.shutdown(verify=True)
+
+def vdi_is_open(vdi):
```

Comment:
Should be named with an action as it does things.

For example this?
```suggestion
def call_vdi_is_open(vdi):
```

Reply:
I modified the function, now a boolean is returned. So I'm not sure we should rename it.